event_management/event_management/src/main/java/com/example/event_management/users/UsersResources.java

```diff
@@ -0,0 +1,43 @@
+package com.example.event_management.users;
+
+import com.example.event_management.jpa.UsersRepository;
+import org.springframework.http.ResponseEntity;
+import jakarta.validation.Valid;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/api/users)
+public class UsersResources {
+
+    private UsersRepository usersRepository;
+
+    public UsersResources(UsersRepository usersRepository){
+        this.usersRepository = usersRepository;
+    }
+    @GetMapping(path = "/get")

```

Comment:
Is this standard adding get in resource path?

```suggestion
    @GetMapping

```